doc/authentication-methods/client-certificate.md

```diff
@@ -33,7 +33,9 @@ certificates. If the `xmpp_addrs` fields are included in the certificate, they a
 When no `xmpp_addrs` are specified, the `cn` (_common name_) field might be
 used, but it is optional. If there is more than one `xmpp_addrs` or both the
 list and `cn` field are empty, the client must include
-authorization entity.
+authorization entity. By default _common name_ is used, however it can be
```

Comment:
@goddammit take a glance please

Reply:
agree, it requires rewording

Reply:
@DenysGonchar could you please sugest refrased version?